settings.yml

```
@@ -39,66 +39,66 @@ footer_donation_cta:
+article3title: "What is Data Science?"
+article3link: "https://www.freecodecamp.org/news/what-is-data-science-what-a-data-scientist-actually-does/"
+article4title: "SQL Subquery in SELECT"
+article4link: "https://www.freecodecamp.org/news/sql-subquery-how-to-sub-query-in-select-statement/"
+article5title: "Rename a File in Linux"
+article5link: "https://www.freecodecamp.org/news/rename-file-linux-bash-command/"
+article6title: "Git Remove Last Commit"
+article6link: "https://www.freecodecamp.org/news/git-remove-last-commit-how-to-undo-a-commit-in-git/"
+article7title: "What is a Java Hashmap?"
+article7link: "https://www.freecodecamp.org/news/what-is-a-java-hashmap/"
+article8title: "CRUD Operations Defined"
+article8link: "https://www.freecodecamp.org/news/crud-operations-crud-definition-in-programming/"
+article9title: "Git Push Local to Remote"
+article9link: "https://www.freecodecamp.org/news/git-push-local-branch-to-remote-how-to-publish-a-new-branch-in-git/"
+article10title: "UX vs UI"
+article10link: "https://www.freecodecamp.org/news/ux-vs-ui-whats-the-difference-definition-and-meaning/"
+article11title: "VLOOKUP in Excel"
+article11link: "https://www.freecodecamp.org/news/vlookup-in-excel-formula-and-example-function/"
+article12title: "Big O Cheatsheet"
+article12link: "https://www.freecodecamp.org/news/big-o-cheat-sheet-time-complexity-chart/"
+article13title: "Git Rename Branch"
+article13link: "https://www.freecodecamp.org/news/renaming-a-git-branch-how-to-rename-the-current-branch-in-git/"
+article14title: "Excel Pivot Table"
+article14link: "https://www.freecodecamp.org/news/how-to-create-a-pivot-table-in-excel/"
+article15title: "What is Alt Text?"
+article15link: "https://www.freecodecamp.org/news/what-is-alt-text-image-alt-text-html-example/"
+article16title: "HTML Dropdown Menu"
+article16link: "https://www.freecodecamp.org/news/html-drop-down-menu-how-to-add-a-drop-down-list-with-the-select-element/"
+article17title: "Python Reverse List"
+article17link: "https://www.freecodecamp.org/news/python-reverse-list-reversing-an-array-in-python/"
+article18title: "Compare Arrays in JS"
+article18link: "https://www.freecodecamp.org/news/how-to-compare-arrays-in-javascript/"
+article19title: "HTML Background Color"
+article19link: "https://www.freecodecamp.org/news/html-background-color-change-bg-color-tutorial/"
+article20title: "Lowercase a String in JS"
+article20link: "https://www.freecodecamp.org/news/how-to-lowercase-a-string-in-javascript-tolowercase-in-js/"
+article21title: "Data Visualization Tools"
+article21link: "https://www.freecodecamp.org/news/data-visualization-tools-guide/"
+article22title: "CSS Selectors Cheatsheet"
+article22link: "https://freecodecamp.org/news/css-selectors-cheat-sheet-for-beginners/"
+article23title: "Sort Dict by Value Python"
+article23link: "https://www.freecodecamp.org/news/sort-dictionary-by-value-in-python/ "
```

Minor issue, but there's a trailing space here. It's been fixed in the trending `english.yaml` file on the CDN, and should probably be removed here, too:

```suggestion
article23link: "https://www.freecodecamp.org/news/sort-dictionary-by-value-in-python/"
```